src/Nethermind/Nethermind.Evm/EOF/EvmObjectFormat.cs

```diff
@@ -261,6 +262,52 @@ public bool ValidateBody(ReadOnlySpan<byte> container, in EofHeader header)
                 return false;
             }
 
+            for (int sectionIdx = 0; sectionIdx < header.CodeSections.Length; sectionIdx++)
+            {
+                SectionHeader sectionHeader = header.CodeSections[sectionIdx];
+                (int codeSectionStartOffset, int codeSectionSize) = (sectionHeader.Start, sectionHeader.Size);
```

Comment:
Construct a tuple just to deconstruct it? Seems weird and potentially wasteful if compiler doesn't optimize it.